prereise/gather/griddata/hifld/const.py

```diff
@@ -118,3 +118,67 @@
     "345": 345,
     "500": 500,
 }
+
+line_reactance_per_mile = {  # per-unit
+    69: 0.0096,
+    100: 0.0063,
+    115: 0.0039,
+    138: 0.0026,
+    161: 0.0021,
+    230: 0.0011,
+    345: 0.0005,
+    500: 0.0002,
+    765: 0.0001,
+}
+line_rating_short = {  # MVA
+    69: 86,
+    110: 181,
```

Comment:
Should this be 100 instead of 110 here?

Reply:
That'll teach me to type manually, rather than copy/paste.

Reply:
Done.